packages/patterns/src/patterns/types.js

```diff
@@ -39,14 +40,14 @@ export {};
  * @typedef {object} PatternKit
  * @property {(specimen: any,
  *             patt: Passable,
- *             check: Checker,
+ *             reject: Rejector,
  *             label?: string|number
- * ) => boolean} checkMatches
+ * ) => boolean} confirmMatches
  * @property {(specimen: any, patt: Pattern) => boolean} matches
  * @property {(specimen: any, patt: Pattern, label?: string|number) => void} mustMatch
  * @property {(patt: Pattern) => void} assertPattern
  * @property {(patt: any) => boolean} isPattern
  * @property {GetRankCover} getRankCover
  * @property {MatcherNamespace} M
- * @property {(specimen: Passable, check?: Checker) => Kind | undefined} kindOf
+ * @property {(specimen: Passable, reject: Rejector) => Kind | undefined} kindOf
```

Comment:
This change is erroneous. The original type seems wrong too, but the `kindOf` in the PatternKit does not have a second argument.

Reply:
Thanks for catching! Already merged, so if you would like to correct, I'm happy to review. Thanks!

Reply:
Working on it, trying to get a type assertion that would catch this

Reply:
Ah. The old `kindOf` has been split in the new `confirmKindOf` and `kindOf`. What's above is mistakenly trying to type what is now `confirmKindOf`, which is not actually exposed.

Reply:
Ugh the whole `patternMatchers.js ` file is `//@ts-nocheck` which is why we didn't get an error. Fantastic...